azure-pipelines.yml

```diff
@@ -0,0 +1,56 @@
+trigger:
+- master
+
+jobs:
+- job: Test
+  pool:
+    vmImage: 'Ubuntu 16.04'
+  strategy:
+    matrix:
+      Node6:
+        node.version: 6.x
+      Node8:
+        node.version: 8.x
+      Node10:
+        node.version: 10.x
+      Node11:
+        node.version: 11.x
+  steps:
+  - task: NodeTool@0
+    displayName: 'Install Node'
+    inputs:
+      versionSpec: $(node.version)
+  - script: npm install
+  - task: Npm@1
+    displayName: 'npm test'
+    inputs:
+      command: custom
+      verbose: false
+      customCommand: test
```

Comment:
Please take a close look at our current Travis-CI. I think we're calling something different than just "npm test".